src/utils/setup/ibc.rs

```diff
@@ -134,4 +136,24 @@ impl TestContext {
 
         Ok(())
     }
+
+    pub fn start_relayer(&mut self) {
```

Comment:
maybe it's a good idea to add params here so we can stop/start on different chains